public/assets/styles/index.scss

```diff
@@ -0,0 +1,79 @@
+@import url(https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,500,600,700%7COpen+Sans:300,400,600,700,800);
+
+
+@import './variables.scss';
+@import './mixins.scss';
+@import '../../../public/assets/styles/signup';
```

Comment:
this path should be `./signup`. please take out the scss extension on lines 4 and 5.